de.metas.vertical.pharma/src/main/java/org/adempiere/impexp/impl/PharmaImportProductInterceptor.java

```diff
@@ -78,6 +85,58 @@ public void onImport(IImportProcess<?> process, Object importModel, Object targe
 			product.setM_Indication_ID(iproduct.getM_Indication_ID());
 		}
 
+		if (iproduct.getM_PharmaProductCategory_ID() > 0)
+		{
+			product.setM_PharmaProductCategory_ID(iproduct.getM_PharmaProductCategory_ID());
+		}
+
 		save(product);
+
+		importPrices(iproduct);
+	}
+
+	private void importPrices(@NonNull final I_I_Product importRecord)
+	{
+		createAPU(importRecord);
+		createAEP(importRecord);
+	}
+
+	private void createAPU(@NonNull final I_I_Product importRecord)
+	{
+		final ProductPriceRequest request = ProductPriceRequest.builder()
+				.price(importRecord.getA01APU())
+				.priceList(importRecord.getAPU_Price_List())
+				.product(importRecord.getM_Product())
+				.validDate(SystemTime.asDayTimestamp())
+				.taxCategory(findTaxCategory(importRecord))
+				.build();
+
+		final ProductPriceImporter command = new ProductPriceImporter(request);
+		command.createProductPrice_And_PriceListVersionIfNeeded();
+	}
+
+	private void createAEP(@NonNull final I_I_Product importRecord)
+	{
+		final ProductPriceRequest request = ProductPriceRequest.builder()
+				.price(importRecord.getA01AEP())
+				.priceList(importRecord.getAEP_Price_List())
+				.product(importRecord.getM_Product())
+				.validDate(SystemTime.asDayTimestamp())
+				.taxCategory(findTaxCategory(importRecord))
+				.build();
+
+		final ProductPriceImporter command = new ProductPriceImporter(request);
+		command.createProductPrice_And_PriceListVersionIfNeeded();
+	}
+
+	private I_C_TaxCategory findTaxCategory(@NonNull final I_I_Product importRecord)
```

Comment:
why not having this method in ITaxDAO.
It shall work out of trx because tax category is master data.